lib/getPostcssResult.js

```diff
@@ -1,28 +1,29 @@
 /* @flow */
 "use strict";
 
-const autoSyntax = require("postcss-html");
+const autoSyntax = require("postcss-syntax");
 const dynamicRequire = require("./dynamicRequire");
 const fs = require("fs");
-const less = require("postcss-less");
-const path = require("path");
 const postcss = require("postcss");
-const safeParser = require("postcss-safe-parser");
-const sass = require("postcss-sass");
-const scss = require("postcss-scss");
-const sugarss = require("sugarss");
+
+const importLazy = require("import-lazy")(require);
 const syntaxes /*: {
   [syntaxName: string]: postcss$syntax,
 }*/ = {
+  sugarss: importLazy("sugarss"),
   css: {
     stringify: postcss.stringify
-  },
-  less,
-  sass,
-  scss,
-  sss: sugarss,
-  sugarss
+  }
 };
+[
+  "less",
+  "sass",
+  "scss",
+  "markdown",
```

Comment:
`markdown` was being removed wasn't it?

Reply:
`postcss-markdown` split from the original `postcss-html`